ntc_templates/templates/mikrotik_routeros_ip_address_print.textfsm

```diff
@@ -7,7 +7,7 @@ Value NETWORK (\S+)
 Value INTERFACE (.*?)
 
 Start
-  ^\s*Flags:\s*(?:X\s*-\s*disabled,)?\s*(?:I\s*-\s*invalid,)?\s*(?:D\s*-\s*(?:DYNAMIC|dynamic))?\s*$$
+  ^\s*Flags:\s*(?:X\s*-\s*(DISABLED|disabled),)?\s*(?:I\s*-\s*(INVALID|invalid),)?\s*(?:D\s*-\s*(?:DYNAMIC|dynamic))?\s*
```

Comment:
why are you removing the end-of-line marker?

Reply:
@jmcgill298 when parsing the CLI through textfsm directly (i.e.: on https://textfsm.nornir.tech/) with the end-of-line marker it would parse without an errors. 

However, when using Netmiko it would barf on that same line until I removed the end-of-line marker. Cannot for the life of me work out why that's the case, as I don't seem to see any unexpected characters in the output from Netmiko.

With end-of-line marker:
```
raise TextFSMError('State Error raised. Rule Line: %s. Input Line: %s'
textfsm.parser.TextFSMError: State Error raised. Rule Line: 14. Input Line: Flags: X - DISABLED, I - INVALID, D - DYNAMIC
```

Without end-of-line marker:
```
[{'num': '0', 'flags': '', 'comment': 'Tech Access'<REDACTED>
```

```
Flags: X - DISABLED, I - INVALID, D - DYNAMIC
Columns: ADDRESS, NETWORK, INTERFACE
 #   ADDRESS            NETWORK         INTERFACE           
;;; Tech Access

'Flags: X - \x02DISABLED, I - \x02INVALID, D - DYNAMIC\nColumns: ADDRESS, NETWORK, INTERFACE\n #   ADDRESS            NETWORK         INTERFACE           \n;;; Tech Access\n
```